docs/12-experimental.md

```diff
@@ -0,0 +1,48 @@
+# Experimental features
+
+"Experimental Features" are cutting-edge additions to Serverpod that are currently under development or testing. These features allow developers to explore new functionalities and provide feedback, helping shape the future of Serverpod. However, they may not be fully stable or complete and are subject to change.
+
+By default, experimental features are disabled. To opt into using them, include the `--experimental-features` flag when running the serverpod command:
+
+```bash
+$ serverpod generate --experimental-features=all
+```
+
+The current options you can pass are:
+
+- all
+- inheritance
```

Comment:
Suggestion: Change this to a table that list the optional available along with a short description about what the experiment is about.